ebi_eva_common_pyutils/assembly/assembly.py

```diff
@@ -11,231 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import os
```

Comment:
This content was moved wholesale to `ebi_eva_common_pyutils/reference/assembly.py`